pkg_resources/__init__.py

```diff
@@ -1693,6 +1693,10 @@ def __init__(self, module):
     def _zipinfo_name(self, fspath):
         # Convert a virtual filename (full path to file) into a zipfile subpath
         # usable with the zipimport directory cache for our target archive
+        while fspath.endswith(os.sep):
+            fspath = fspath[:-1]
```

Comment:
Could these two lines be `fspath = fspath.rstrip(os.sep)`?

Reply:
Done